tests/HelpersTest.cpp

```diff
@@ -998,6 +998,10 @@ TEST_CASE("Types: TypeName", "[helpers]") {
     CHECK(umapName == "[INT,[TEXT,FLOAT]]");
 
     vclass = CLI::detail::classify_object<std::atomic<int>>::value;
+    CHECK(CLI::detail::object_category::wrapper_value == vclass);
+
+    std::string atomic_name = CLI::detail::type_name<std::atomic<int>>();
+    CHECK(atomic_name == "INT");
```

Comment:
@phlptp This actually reports `CLI::detail::object_category::other` and "TEXT" on some compilers (older clang and GCC). Any quick thoughts on if that is easily fixable / important?

Reply:
Things got a little weird with older compilers and atomics and the template checks.   I think it still works when actually used just in a roundabout fashion through the stream operators and I am not sure the effort to fix the template names for those compilers is worth it.

Reply:
Okay, we can just check for both then.